src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -20,8 +20,8 @@
 import org.opensearch.common.network.NetworkService;
 import org.opensearch.common.settings.Settings;
 import org.opensearch.common.util.PageCacheRecycler;
-import org.opensearch.discovery.PluginRequest;
-import org.opensearch.discovery.PluginResponse;
+import org.opensearch.discovery.InitializeExtensionsRequest;
+import org.opensearch.discovery.InitializeExtensionsResponse;
```

Comment:
The changes in this file look good but you are importing changed class names in the `org.opensearch.discovery` package.  How does this class access those dependencies?  Is there another PR against a different repository that this is linked to?

Reply:
Equivalent PR on OpenSearch : https://github.com/opensearch-project/OpenSearch/pull/3951